manifests/admin.pp

```diff
@@ -158,14 +174,16 @@
     user_list => ["%${admin_group}"],
     runas     => 'ALL',
```

Comment:
I feel like we should restrict this to `root` since we're adding in the SELinux context material.

Perhaps make it another configurable option with documentation in the comments?

Reply:
Added the options for both auditors and administrators.